REMAINING.PUBLIC.md

```diff
@@ -0,0 +1,60 @@
+### Remaining public aliases
+
+annotation_reader
+security.access.decision_manager
+security.acl.cache
+security.acl.provider
+security.encoder_factory
+security.password_encoder
+session.handler
+session.storage
+templating.loader
```

Comment:
IMHO, all theses services (above) could be private. except the `security.password_encoder`  (for use in ContainerAware classes)